packages/@o3r/rules-engine/src/engine/operator/operators/date-based.operators.ts

```diff
@@ -16,6 +17,74 @@ export const inRangeDate: Operator<Date, [DateInput, DateInput], DateInput> = {
   validateRhs: isValidDateRange
 };
 
+/**
+ * Validate if the provided operatorFactValues contain 'o3rCurrentTime'
+ * @param value the value
+ */
+export function isValidCurrentTimeValue(value: unknown): value is Record<string, Facts> {
```

Comment:
I think the name of the function is unclear, it doesn't check if the current time value is valid, it just checks if it contains the key